dynatrace-oneagent-operator/questions.yml

```diff
@@ -38,6 +38,43 @@ questions:
   required: true
   group: "Agent Configuration (REQUIRED)"
 
+  #################### Use custom resource settings ###################
+
+- variable: use_custom_resource_settings
```

Comment:
I'd suggest limits instead of settings, since "custom resource" is already a thing.

Reply:
Changed